guides/common/modules/con_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc

```diff
@@ -0,0 +1,4 @@
+[id="con_Configuring_{smart-proxy-context}_Server_with_Custom_SSL_Certificates_for_Load_Balancing_without_Puppet_{context}"]
```

Comment:
Thanks for all the input so far! Everything is ready for review now @adamlazik1 

I think this is the last discussion-worthy item. Either keep it as is or come up with a better title for the procedure in `guides/common/modules/proc_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc`.

Reply:
> Thanks for all the input so far! Everything is ready for review now @adamlazik1

The review is underway :rocket: 

>I think this is the last discussion-worthy item. Either keep it as is or come up with a better title for the procedure in guides/common/modules/proc_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc.

I think it's probably fine keeping it as it is.